### DISCUSSION TITLE
s/fv379mgibovyky73kcommq

### DISCUSSION THREAD

**Discussion by giscus** [ops-coffee/blog#16] [Announcements]
# s/fv379mgibovyky73kcommq

Proxy：简单小巧又强大好用的代理系统

https://blog.ops-coffee.cn/s/fv379mgibovyky73kcommq

**Comment by lipeibin** [comment]
大佬，有开源不？

**Reply by ops-coffee** [reply]
公司项目都不开源，介绍下技术思路，具体时间起来并不难

**Comment by seekplum** [comment]
实现的前提是 配置了 DNS，可以在访问 内网域名 时解析到 对应Nginx 服务对吗？
